### ISSUE TITLE
Prompts unreadable on light themes

### ISSUE THREAD

**Issue by qtlis** [tmattio/spin#88]
Presumably thanks to the text colour being force-set to white, the prompts are unreadable in a terminal using a light theme:

Konsole/Yakuake with Linux colours theme for reference
![Screenshot_20201006_230138](https://user-images.githubusercontent.com/15247429/95259852-1fca5580-0828-11eb-9f5c-574373c88ab8.png)

…and with with some default light themes
![Screenshot_20201006_230156](https://user-images.githubusercontent.com/15247429/95260113-7df73880-0828-11eb-8522-b9745c1c367a.png)
![Screenshot_20201006_230212](https://user-images.githubusercontent.com/15247429/95260115-7e8fcf00-0828-11eb-88f5-a214eddcf849.png)
![Screenshot_20201006_230222](https://user-images.githubusercontent.com/15247429/95260117-7f286580-0828-11eb-90ab-cd8d9e028974.png)

LXTerminal with Solarized Light
![Screenshot_20201006_230817](https://user-images.githubusercontent.com/15247429/95260455-083f9c80-0829-11eb-9159-dff49fa2d854.png)

**Comment by citizen428** [comment]
Some terminals export a variable called `COLORFGBG` (e.g. iTerm2, rxvt), other xterm compatible terminals rely on an escape sequence:

```sh
$ echo $COLORFGBG
15;0

$ printf '\033]11;?\007'
11;rgb:3e/3e/3e
```

One could probably wrap this up in some detection code and then change the output color based on this.

**Comment by tmattio** [comment]
Thanks for opening the issue @qtlis and thanks for the tip @citizen428!

Will look into it as soon as I have some spare time. The fix will most likely have to be implemented in [inquire](https://github.com/tmattio/inquire), since that where the prompts are handled.

**Comment by citizen428** [comment]
@tmattio If I have time — which wasn't really the case these past few months as you have noticed — I'll take a look at inquire and see what I can come up with.

**Comment by tmattio** [comment]
@citizen428 Sure, let me know if I can help with anything if you end up looking into it 🙂

**Comment by tmattio** [comment]
This should be fixed by #111, which upgraded to the latest version of `inquire`. The prompts are now bold only and use the default color of the terminal.

I'll cut a release of Spin soon, so the next version will be usable on light terminals 🙂
